tests/unit/sys/test_expressionparser.cxx

```diff
@@ -49,7 +45,7 @@ class BinaryGenerator : public FieldGenerator {
     return std::make_shared<BinaryGenerator>(args.front(), args.back());
   }
 
-  BoutReal generate(Position pos) {
+  BoutReal generate(const Context& pos) override {
```

Comment:
If we rename `Position` to `Context` - should `pos` be renamed to `con` or `context`

Reply:
Yes probably, but that was harder to search and replace.